powa/static/js/components/Grid.vue

```diff
@@ -67,23 +65,38 @@
           </thead>
         </template>
         <!-- This template looks for headers with formatters and executes them -->
-        <template
-          v-for="header in headers.filter((header) =>
-            header.hasOwnProperty('formatter')
-          )"
-          #[`item.${header.value}`]="{ value }"
-        >
-          <query-tooltip
-            v-if="header.type == 'query'"
-            :key="header.value"
-            :value="value"
-          ></query-tooltip>
-          <template v-else-if="header.type == 'bool'">
-            <span :key="header.value" v-html="header.formatter(value)"></span>
-          </template>
-          <template v-else>
-            {{ header.formatter(value) }}
-          </template>
+        <template #item="{ item }">
+          <tr>
+            <td
+              v-for="header in headers.filter((header) =>
+                header.hasOwnProperty('formatter')
+              )"
+              :key="(header.value, item.id)"
+              :class="[header.cellClass, 'text-' + header.align]"
+            >
+              <query-tooltip
+                v-if="header.type == 'query'"
+                :key="header.value"
+                :value="
+                  header.formatter(
```

Comment:
What about changing the `formatter` signature to `(item, header)` and build the url in the formatter?